lnwire/message.go

```diff
@@ -109,7 +111,12 @@ func makeEmptyMessage(command uint32) (Message, error) {
 	case CmdErrorGeneric:
 		msg = &ErrorGeneric{}
 	default:
-		return nil, fmt.Errorf("unhandled command [%d]", command)
+		// TODO(mkl): maybe it is better to put messages directly here.
```

Comment:
I agree with this `TODO`. We can avoid those gymnastics to avoid the import cycle if all the routing messages are put within the `lnwire` page. Additionally, we may want to create a nested sub-package within the `lnwire` package dedicated to housing the routing messages of a particular strategy.